anap1003/assignment1/Q1.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment1;
+
+import java.util.*;
+
+/**
+ *
+ * @author Ana
+ */
+public class Q1 {

```

Comment:
Try to get into the habit of providing useful javadoc comments that explain the classes you write.